lib/api/community/leaderboard.ml

```diff
@@ -11,6 +11,10 @@ let get game domain send =
   | None -> Lwt.return None
 ;;
 
+let get game domain send = get_available_leaderboards game domain send
+[@@deprecated "Use the more clearly named 'get_available_leaderboards' instead"]
```

Comment:
For some reason this isn't working. Still debugging.

Reply:
Fixed. Turns out you can't use the `[@@deprecated ...]` attribute in `.ml` files only in interface files.